product_owners.md

```diff
@@ -0,0 +1,27 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. Documentation of our team's [work cycles](/work_cycles.md) and [technical liaison role](/technical_liaisons.md) should give product owners useful context for this collaboration.
+
+The exact practices of each product owner will vary according to the application and the
+individuals themselves. Generally, product owners will work to:
+
+* Communicate with other stakeholders and users as-needed, either ad-hoc as a committee.
```

Comment:
I wonder if it makes sense to rework lines 10 and 13 to say that it's helpful for a PO to have a general sense of top stakeholder priorities on an ongoing basis.

Reply:
@hackartisan Take another look?